test/fixtures/javascript-prop-types/main.js

```diff
@@ -0,0 +1,8 @@
+import fs from "fs";
+import process from "process";
+import * as TopLevel from "./toplevel.js";
+
+const sample = process.argv[2];
+const json = fs.readFileSync(sample);
```

Comment:
It doesn't seem like this is necessary, is it?  All you want is for `toplevel.js` to be imported correctly.  You don't have to copy the same file, then, either.

I'm fine to merge this if you don't want to change it anymore.

Reply:
Yah, this is in progress. I was actually planning on having it read the json and validate it against the input prop-types.